core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -382,14 +382,14 @@ class KafkaConfigTest {
 
     props.put(KafkaConfig.InterBrokerProtocolVersionProp, "0.8.2.0")
     // We need to set the message format version to make the configuration valid.
-    props.put(KafkaConfig.LogMessageFormatVersionProp, "0.8.2.0")
+    props.put(KafkaConfig.LogMessageFormatVersionProp, "0.8.2")
```

Comment:
What was the reason for changing this?

Reply:
It's not a valid value actually, instead `0.8.2` is.

Reply:
Haha, good point